src/main/java/baseball/domain/Answer.java

```diff
@@ -0,0 +1,58 @@
+package baseball.domain;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static baseball.utils.ExceptionMessage.*;
+
+public class Answer {
+
+    private static final int ANSWER_DEFAULT_SIZE = 3;
+    private static final int MIN_BALL_SIZE = 1;
+    private static final int MAX_BALL_SIZE = 9;
+    private final List<Integer> numbers;
+
+    public Answer(List<Integer> numbers) {
+        validateAnswer(numbers);
+        this.numbers = numbers;
+    }
+
+    public List<Integer> getNumbers() {
+        return this.numbers;
+    }
+
+    private List<Integer> validateAnswer(List<Integer> numbers) {
+        List<Integer> playerNum = new ArrayList<>();
+        if (AgainSize(numbers)) {
+            throw new IllegalArgumentException(INPUT_ERROR_SIZE.getMessage());
+        }
+        if (AgainNumber(numbers)) {
+            throw new IllegalArgumentException(INPUT_ERROR_AGAIN.getMessage());
+        }
+        if (AgainRange(numbers)) {
+            throw new IllegalArgumentException(INPUT_ERROR_RANGE.getMessage());
+        }
+
+        for (Integer number : numbers) {
+            playerNum.add(number);
+        }
+        return playerNum;
+    }
+
+    private boolean AgainSize(List<Integer> numbers) {
+        return numbers.size() != ANSWER_DEFAULT_SIZE;
+    }
+
+    private boolean AgainNumber(List<Integer> numbers) {
+        Set<Integer> duplicate = new HashSet<>(numbers);
```

Comment:
set 안에는 중복되지 않은 값이 들어갈텐데 이름을 `duplicate`로 해놓으면 모순이 있는 것 같아요.
`중복되지 않은` 이라는 뜻이 들어가게 수정하면 좋을 것 같아요. 

```suggestion
        Set<Integer> deduplicatedNumbers = new HashSet<>(numbers);
```

Reply:
수정하겠습니다!!!!!